frontend/packages/dev-console/src/components/custom-resource-list/CustomResourceList.tsx

```diff
@@ -72,7 +72,6 @@ const CustomResourceList: React.FC<CustomResourceListProps> = ({
           rowFilters={rowFilters}
           data={listItems}
           textFilter={textFilter}
-          hideNameFilter={false}
           hideLabelFilter
```

Comment:
Hide the toolbar here? That seems to be the intent of the current code.

```suggestion
          hideToolbar
```

Reply:
`hideNameFilter={false}` looks like they only want the name filter.`hideToolbar` would hide everything.

Reply:
Thanks, I misread.